e2e-tests/README.md

```diff
@@ -25,11 +25,18 @@ rest of Socorro so you're not mixing requirements:
 	$ # your dev virtualenv.
 	$ pip install -r requirements.txt
 
-An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads].
+An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads]. Note that on Linux, you don't need to downgrade your Firefox install in order to use an older version.
```

Comment:
I'm surprised by that. What version of Firefox / Selenium are compatible now? I'm testing on Ubuntu 16.04 with Firefox 50.0.1 and Selenium 3.0.2, and I get this error: 

```bash
$ py.test --driver="Firefox" --base-url="http://localhost:8000" -x tests/test_search.py
======================================= test session starts =======================================
platform linux2 -- Python 2.7.12, pytest-3.0.4, py-1.4.31, pluggy-0.4.0
[...]
WebDriverException: Message: Can't load the profile. Profile Dir: /tmp/tmpAg7sH3 If you specified a log_file in the FirefoxBinary constructor, check it for details.
```